File: Baekjoon/S5/C#/S5_수 정렬하기 2.cs
https://www.acmicpc.net/submit/2751

using System;
using System.Text;

namespace Baekjoon
{
    class Program
    {
        static void Main(string[] args)
        {
            StringBuilder sb = new StringBuilder();
            int[] arr = new int[int.Parse(Console.ReadLine())];
            for (int i = 0; i < arr.Length; i++)
            {
                arr[i] = int.Parse(Console.ReadLine());
            }

            Array.Sort(arr);
            for (int i = 0; i < arr.Length; i++)
            {
                sb.Append(arr[i] + "\n");
            }

            Console.WriteLine(sb);
        }
    }
}

File: Baekjoon/S5/JAVA/S5_수 정렬하기 2.java
//https://www.acmicpc.net/problem/2751

import java.io.*;
import java.util.*;

public class Main{
    public static void main(String args[]) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int[] arr = new int[n];
        for(int i = 0; i < n; i++)
            arr[i] = Integer.parseInt(br.readLine());
        
        Arrays.sort(arr);
        
        StringBuilder sb = new StringBuilder();
        for(int i = 0; i < n; i++)
            sb.append(arr[i]+"\n");
        
        System.out.print(sb);
    }
}